src/main/java/Calculations/GPACalculation.java

```diff
@@ -0,0 +1,167 @@
+package Calculations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.Math;
+
+/**
+ * Construct GPACalculations that consist of different methods for various functionality ranging from computation
+ * of percentage to GPA, converting list marks to list of GPA marks, computing the average of marks,
+ * computation of percentage to lettered Grade, converting list marks to list of lettered Grades, and overall GPA
+ * consisting of GPA and weight. GPA scale and lettered Grade are representive of UofT grading scheme.
+ */
+
+public class GPACalculation {
+
+    /**
+     * Takes a double of mark in terms of percentage and converts to double in terms of GPA point scale
+     * @param percentage is a double that represents the mark in a course
+     * @return a double of GPA value
+     */
+
+    public static double computationGPA(double percentage){
```

Comment:
consider renaming to percentageToGPA to comply with other naming decision (percentageToGrade)